### DISCUSSION TITLE
about page image title and alt text

### DISCUSSION THREAD

**Discussion by kbvernon** [quarto-dev/quarto-cli#3009] [Q&A]
Is there a way to specify the image title and alt text in the yaml of an about page? The documentation for the about page suggests you can only change image width and shape, but maybe there's information somewhere else about working with images linked in the yaml?

Thanks!

**Comment by dragonstyle** [comment]
Right now there doesn't look like a way to specify this, though there should be. I'm opening an issue to track this and will try to get that fixed soon!

**Reply by kbvernon** [reply]
Awesome! Thanks for looking into this!

**Comment by FlukeAndFeather** [comment]
Looks like alt-text is now supported in v1.3, but what about captions? Something equivalent to `![some caption here](about-image.jpg)`. Like an `img-caption` option?
